exercises/pascals-triangle/pascals_triangle.py

```diff
@@ -1,10 +1,10 @@
-def triangle():
+def triangle(row_count):
     pass
 
 
-def is_triangle():
+def is_triangle(triangle):
```

Comment:
Maybe more appropriate name for the parameter will be `triangle_rows_list` or something similar. 
`triangle` can be mistaken by an instance of class Triangle for example.
What do you think?

Reply:
That certainly makes sense. I've updated the parameter name.